src/main/java/Task.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * A Task class is a general class that represents a Task.
+ * It has a Description, and boolean whether it is done or not.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
Just a suggestion: I used enum to create TaskType and use Tasktype as part of the printing (further leverage on Liskov Sub Principle)